lib/keys.js

```diff
@@ -41,7 +41,20 @@ Keys.prototype.create = function(fn) {
 
     data[key] = true;
     keys.writeFile(data, function(err) {
-      fn(err, key);
+      if(err){
+        if(err.code == 'ENOENT' && err.path == '.dpd/keys.json'){
+          console.error('Path: '+err.path+' not found, creating new one.');
+          // TODO: Could ask with prompt?
```

Comment:
Should it prompt "Do You want to create a new directory '.dpd'?" if "No such file or directory"?